src/core/client/stream/events.ts

```diff
@@ -622,3 +622,17 @@ export const CancelAccountDeletionEvent = createViewerNetworkEvent<{
     code?: string;
   };
 }>("cancelAccountDeletionEvent");
+
+/**
+ * This event is emitted when the viewer changes the moderation
+ * queue sort.
+ */
+export const QueueSortChangedEvent = createViewerNetworkEvent<{
+  success: {
+    sortOrder: string;
+  };
+  error: {
+    message: string;
+    code?: string;
+  };
+}>("queueSortChangedEvent");
```

Comment:
use `createViewerEvent` instead of `createViewerNetworkEvent` as this action does not cause a network request.